### PR TITLE
Use our modified stacks.json when building Che

### PR DESCRIPTION
**Pull request by l0rd**
This is needed to use root-free che stacks.

@ibuziuk please review

### CODE REVIEW

File: build_che.sh [hunk 1]
```diff
@@ -16,7 +16,9 @@ scl enable rh-maven33 'mvn -B clean install'
 cd ..
 
 git clone -b ${GIT_BRANCH} ${GIT_REPO}
-cd che 
+cd che
+# Use our modified stacks
+mv ./ide/che-core-ide-stacks/src/main/resources/stacks.json.centos \ 
+ ./ide/che-core-ide-stacks/src/main/resources/stacks.json
```
Comment (ibuziuk) [inline build_che.sh]: Should not we revert stack files after the build (similar to Dockerfiles) ?
 ```
mv Dockerfile Dockerfile.alpine
cp Dockerfile.centos Dockerfile
./build.sh ${CHE_IMAGE_TAG}
mv Dockerfile.alpine Dockerfile
```

Comment (l0rd) [reply build_che.sh]: Good point but we do not revert Dockerfile in the jenkins job https://github.com/kbsingh/build-run-che/blob/master/cico_build.sh#L28. That is done in rh-che repository and yes we need to update that too.

For consistency I would not revert stacks.json neither but I've changed the PR to use `cat` to do the replacement as it is done for the Dockerfile.

Comment (ibuziuk) [reply build_che.sh]: @l0rd but it means that if I work with che I will need to revert this change manually. One more point - Should not we build `che-core-ide-stacks` component in the script ?

**Comment by ibuziuk** [comment]
@l0rd sorry, do not pay attention to my last comments - I thought I review "rh-che" repo :/

**Review by ibuziuk — APPROVED** [review]

File: build_che.sh (hunk 1 above)
Comment (l0rd) [reply build_che.sh]: This script is for jenkins job usage only so a developer should not use it.
`che-core-ide-stacks` component is already build when we build from the root `che` directory isn't it?

**Pull request merged**